docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md

```diff
@@ -79,7 +79,12 @@ az account list --query "[?isDefault]"
 
 # get a subscription that contains search words or phrases
 az account list --query "[?contains(name,'search phrase')].{SubscriptionName:name, SubscriptionID:id, TenantID:tenantId}" --output table
+```
+
+---
```

Comment:
remove line

Reply:
Done. The main reason I use a line is because I get a warning, but I have removed the extra line.

Reply:
@v-abdriaz , The extra `---` line is still there, so I edited your branch and removed it.